builds/latest-nuget-build-test.yml

```diff
@@ -2,17 +2,17 @@ trigger: none
 
 pr: none
 
-# Pipeline trigger - this pipeline will run whenever the 'DacFx Nuget - Nightly' pipeline finishes
+# Pipeline trigger - this pipeline will run whenever the 'DacFx-Official' pipeline finishes
 resources:
   pipelines:
   - pipeline: 'latest-nuget-build-test'
-    source: 'DacFx Nuget - Nightly'
+    source: 'DacFx-Official'
     trigger: true
 
 variables:
   solution: '**/*.sln'
   configuration: 'Debug'
-  dacFxPackagePattern: 'Microsoft.SqlServer.DacFx*-preview.nupkg'
+  dacFxPackagePattern: 'Microsoft.SqlServer.DacFx*.nupkg'
```

Comment:
Are we going to retire Preview NuGet in future? Or this change will fetch the preview nuget only?

Reply:
No the DacFx-Official pipeline by default only generates non-preview Nuget in the scheduled runs. This pipeline is used only by the SDK to test against latest DacFx changes.